app/src/main/AndroidManifest.xml

```diff
@@ -1,21 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="assembly.giraff" >
+    <uses-permission android:name="android.permission.internet"/>
```

Comment:
This should be:

<uses-permission android:name="android.permission.INTERNET"/>